src/pages/test-page.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react"
+import { Layout } from "../components/Layout/Layout"
+import SettingsModal from "../components/SettingsModal/SettingsModal"
+
+const TestPage = () => {
+  const [modalActive, setModalActive] = useState<boolean>(false)
+
+  const toggleModal = () => {
+    // eslint-disable-next-line no-console
+    console.log("toggle")
```

Comment:
I know this is only for testing, but before we merge we should remove the ignore and the console.log 😄

Reply:
+1

Reply:
Small question actually, why are we committing a test page? 🤔
Shouldn't we just be importing the use of the modal into the correct component?

Reply:
@ksamson2 good point actually 😂